### DISCUSSION TITLE
Some questions about the quantitative protein process

### DISCUSSION THREAD

**Discussion by xiaosiruwei** [vdemichev/DiaNN#991] [Q&A] [answered]
Why is protein quantification based on groups? And what does gene goups mean
Is there two methods for protein quantification? One is TopN, the other is MaxLFQ, and the output results both display the quantitative results of these two methods? The MaxLFQ method is based on [doi. org/10.1074/mcp.M113.031591](url) a quantitative algorithm for this biased paper?

**Comment by vdemichev** [comment, marked as answer]
One also has quantities calculated with gene-specific peptides reported in Genes.MaxLFQ.Unique.

The method is based on MaxLFQ algorithm as introduced by Juergen Cox and colleagues with some modifications.

**Reply by xiaosiruwei** [reply]
Hi, in the scoring of putative elution peaks by DIA-NN, the Supplementary Information states that there are characteristics (73 total), but the array scoring size in the code is 110. What's going on?

**Reply by vdemichev** [reply]
Newer versions have more scoring characteristics

**Reply by xiaosiruwei** [reply]
Thank you, I will be looking forward to it very much！
